native/packages/react-native-bpk-component-pagination-dots/readme.md

```diff
@@ -56,7 +52,6 @@ export default class App extends Component {
 
 | Property            | PropType                  | Required | Default Value |
 | -----------         | ------------------------- | -------- | ------------- |
-| accessibilityLabel  | oneOfType(string, func)   | true     | -             |
```

Comment:
We discussed the removal of `accessibilityLabel` separately, TLDR is that this component is purely a visual indicator and as such does not require interaction or any a11y considerations.